.travis.yml

```diff
@@ -8,8 +8,11 @@ node_js:
   - "iojs-v3"
   - "4"
   - "5"
+  - "6"
 env:
   - BROWSER=1
```

Comment:
I just saw that! My apologies! Just catching up on the great work you've done just in the past few days!